DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/KeywordCampaignsServlet.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import java.io.IOException;
+import java.io.PrintWriter;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
+// manages the keyword campaigns in datastore
+@WebServlet("/keyword-campaigns")
+public class KeywordCampaignsServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Can you add simple tests for these?

Reply:
So these functions don't return anything, is it ok if we test it by running the web server? Or if billing gets set up, we can deploy and test it easily then

Reply:
Even though they don't return anything, aren't they modifying the response? If so, it would be good to test a simple request and check that the response is expected and/or the class is modified properly.

And billing should be setup now. Could you try deploying and linking the site in the README?